cpu/stm32l1/periph/i2c.c

```diff
@@ -486,6 +496,13 @@ void i2c_poweroff(i2c_t dev)
             I2C_0_CLKDIS();
             break;
 #endif
+#if I2C_1_EN
+        case I2C_1:
+            while (I2C_0_DEV->SR2 & I2C_SR2_BUSY);
+
```

Comment:
I'm wondering about these newlines (here and in line 495). I know, it's not your fault ;-)